tests/foreman/ui/test_packages.py

```diff
@@ -132,6 +136,34 @@ def test_positive_check_package_details(self):
                 ]
             )
 
+    @tier2
+    @skip_if_bug_open('bugzilla', 1394390)
+    def test_positive_check_custom_package_details(self):
```

Comment:
This test covers `1384673` that is related to `Content -> Packages` when uploading custom packages, while `1394390` is a blocker as it does not allow to upload custom packages and `Manage packages` related to `Content -> Repositories`. There is separate test for uploading  custom packages in repository test-suite, so there is no need to cover that in this test.